server/controllers/userInputController.ts

```diff
@@ -164,6 +171,10 @@ const createUserInputController = (
       const { userInputId } = req.params;
 
       logger.debug('Updating title for userInput:', { userInputId, title });
+      // Check for authorization header. If not provided, throw 401 error
+      if (!req.headers.authorization) {
```

Comment:
The check for auth header in this controller is redundant here as it is already checked in the auth middleware.